pcweb/pages/docs/tutorial/final_app.py

```diff
@@ -32,17 +32,31 @@ def answer(self):
     # Add to the answer as the chatbot responds.
     answer = ""
     self.chat_history.append((self.question, answer))
+    # After adding the question into the chat history,
+    # we clear the question and yield such that
+    # the frontend updates now instead of sending an event.
```

Comment:
I think we can just add one comment here saying `# Clear the question input` and then right above the `yield` add the other comment saying `# Yield here to clear the frontend input before continuing`

(Realized this comment should apply to the previous page also)